R/with-indices.R

```diff
@@ -88,4 +88,10 @@ torch_cross_entropy_loss <- function(
   target <- target$sub(1L, 1L)
   .torch_cross_entropy_loss(self = self, target = target, weight = weight,
                             reduction = reduction, ignore_index = ignore_index)
-}
\ No newline at end of file
+}
+
+torch_sort <- function(self, dim = -1L, descending = FALSE, stable = TRUE) {
+  out <- .torch_sort(self, dim, descending = FALSE, stable )
```

Comment:
This doesn't respect the `descending` argument passed to `torch_sort()`.

Reply:
That is true , thanks @hsbadr .